server/database/User.js

```diff
@@ -0,0 +1,105 @@
+var mongoose = require('mongoose');
+
+var UserSchema = new mongoose.Schema({
+    name: String,
+    email: String,
+    password: String,
+    link: String,
+    displayName: String,
+    isTeacher: Boolean,
+    sentMessageIDs: [String],
+    recievedMessageIDs: [String],
+    groupIDs: [String],
+    active: Boolean,
+    auth: {
+        token: String
+    }
+})
+
+var UserModel = mongoose.model('User', UserSchema );
+
+
+class User {
+
+    constructor(userIn){
+        this.validateUser(userIn)
+        if (userIn.isTeacher ){
```

Comment:
Why is this if statement here

Reply:
I actually meant to delete that whole folder